firmware/config/boards/common_make.sh

```diff
@@ -25,6 +25,23 @@ fi
 cp build/rusefi.hex deliver/
 cp build/rusefi.bin deliver/
 
+# bootloader
+if [ $USE_OPENBLT = "yes" ]; then
+  make openblt
+  [ -e build/openblt_$PROJECT_BOARD.hex ] || { echo "FAILED to compile OpneBLT by $SCRIPT_NAME with $PROJECT_BOARD"; exit 1; }
+  rm -f deliver/openblt_$PROJECT_BOARD.dfu
+  echo "$SCRIPT_NAME: invoking hex2dfu for OpenBLT"
+  if uname | grep "NT"; then
+   chmod u+x ../misc/encedo_hex2dfu/hex2dfu.exe
+   ../misc/encedo_hex2dfu/hex2dfu.exe -i build/openblt_$PROJECT_BOARD.hex -o deliver/openblt_$PROJECT_BOARD.dfu
+  else
+   chmod u+x ../misc/encedo_hex2dfu/hex2dfu.bin
+   ../misc/encedo_hex2dfu/hex2dfu.bin -i build/openblt_$PROJECT_BOARD.hex -o deliver/openblt_$PROJECT_BOARD.dfu
+  fi
```

Comment:
can we have combined .dfu file so that my process does not change?

wait, would that mean two different DFU files - one for USB DFU (bl+rusefi) and another one for CAN update (only re?)

Reply:
Yes. Two: One for (initial) flashing of board (OpenBLT+RusEFI combined). And second for RusEFI update: dfu or srec, dependent on method of update.
OpenBLT tools expect *.srec files, not dfu. But we can learn it to understand dfu too.